tests/ORM/HashidableTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Knp\DoctrineBehaviors\Tests\ORM;
+
+use Knp\DoctrineBehaviors\Tests\AbstractBehaviorTestCase;
+use Knp\DoctrineBehaviors\Tests\Fixtures\Entity\HashidableEntity;
+
+final class HashidableTest extends AbstractBehaviorTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function testAutoSettingHashId(): void
+    {
+        $entity = new HashidableEntity();
+
+        $this->entityManager->persist($entity);
+        $this->entityManager->flush();
+
+        $id = $entity->getId();
+        $this->assertNotNull($id);
+
+        $hashId = $entity->getHashId();
+        $this->assertNotNull($hashId);
+        $this->assertNotEmpty($hashId);
```

Comment:
The hash is value should be asserted more precisely than just not `null`. Just to be sure the hashable works, and it's not just generated value by Doctrine.

Reply:
@TomasVotruba I added two more tests. Currently some checks from this repository doesn't work. But I think it's b/c they also don't work on the master branch. Locally everything works fine.